### PR TITLE
Feature/monitoring-certificates

### PR DESCRIPTION
**Pull request by Talits**

### CODE REVIEW

File: ansible/ansible-cfg [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# config file for ansible -- http://ansible.com/
```
[the remark this answers is not in the document]
Comment (caruccio) [reply ansible/ansible-cfg]: Should use /ansible/ansible.cfg instead create a new one.

File: ansible/roles/defaults/main.yml [hunk 2]
```diff
@@ -18,6 +18,7 @@ getupcloud_api_django_default_logger: "production"
 kubernetes_service_account_token_file: "/run/secrets/kubernetes.io/serviceaccount/token"
 getupcloud_api_allow_session_authentication: False
 getupcloud_oc_version_number: "3.9.0"
+getupcloud_openshift_release: "3.9.0"
```
[the remark this answers is not in the document]
Comment (caruccio) [reply ansible/roles/defaults/main.yml]: ☝️

**Comment by Talits** [comment]
comments?

File: ansible/roles/getup-certificate-checker/files/dependencies/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,14 @@
+FROM openshift/origin-ansible
+MAINTAINER TALITA BERNARDES PEREIRA
+
+USER root
```
Comment (caruccio) [inline ansible/roles/getup-certificate-checker/files/dependencies/Dockerfile]: Is it really necessary to to change to `root` here?

Comment (Talits) [reply ansible/roles/getup-certificate-checker/files/dependencies/Dockerfile]: yes - because the image base

**Comment by Talits** [comment]
I'll use jq and fix ansible.cfg - wait pls

File: ansible/roles/getup-certificate-checker/files/dependencies/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,13 @@
+FROM openshift/origin-ansible:${TARGET_OPENSHIFT_RELEASE}
+MAINTAINER TALITA BERNARDES PEREIRA
+
+USER root
+RUN yum install jq -y && mkdir -p /cert
```
Comment (caruccio) [inline ansible/roles/getup-certificate-checker/files/dependencies/Dockerfile]: missing `yum clean all` at the end of the `RUN` directive

File: ansible/ansible-cfg (hunk 1 above)
[the remark this answers is not in the document]
Comment (caruccio) [reply ansible/ansible-cfg]: I mean, there is already a ansible.cfg file at `/ansible`. Why should we have to identical files?

[the remark this answers is not in the document]
Comment (Talits) [reply ansible/ansible-cfg]: because the ansible not found in the container - fail in any forms

**Comment by Talits** [comment]
[Monitoração dos certificados do cluster](https://trello.com/c/dU9ahy9M/58-monitora%C3%A7%C3%A3o-dos-certificados-do-cluster)

**Pull request merged**